s3/pom.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.yahoo.ycsb</groupId>
+        <artifactId>binding-parent</artifactId>
+        <version>0.4.0-SNAPSHOT</version>
+        <relativePath>../binding-parent</relativePath>
+    </parent>
+  
+  <artifactId>s3-binding</artifactId>
+  <name>S3 Storage Binding</name>
+  <packaging>jar</packaging>
+
+  <dependencies>
+    <dependency>
+	<groupId>com.amazonaws</groupId>
+	<artifactId>aws-java-sdk-s3</artifactId>
+	<version>${s3.version}</version>
+    </dependency>
+
+    <dependency>
+      <groupId>com.yahoo.ycsb</groupId>
+            <artifactId>core</artifactId>
+            <version>${project.version}</version>
+            <scope>provided</scope>
+    </dependency>
+   </dependencies>
+</project>
```

Comment:
Please copy the checkstyle config from the mongodb binding's pom and fix any errors it finds.